src/lib/Recorder.ts

```diff
@@ -1,55 +1,64 @@
+/* eslint-disable max-statements */
+/* eslint-disable max-len */
 import PipeSource from './PipeSource';
 
 type TRecorderOptions = {
 	bufferSize?: number;
 	sampleRate?: number;
 	channelCount?: number;
+	publishFrequency?: number;
 };
 
 const defaultOptions: TRecorderOptions = {
 	bufferSize: 1024,
 	sampleRate: 44100,
 	channelCount: 1,
+	publishFrequency: 200,
 };
 
 export default class Recorder extends PipeSource {
 	private readonly options: TRecorderOptions;
 	private isRecording: boolean = false;
 	private micStream: MediaStream;
-	private mediaRecorder: MediaRecorder;
+	private animationID: number;
```

Comment:
camelCase means only capitalizing the first letter of abbreviations. so "Id" and e.g. "Json".
so this should be called "animationId".